tabs-tabs-tabs/tabs.js

```diff
@@ -70,5 +70,49 @@ document.addEventListener("click", function(e) {
     });
   }
 
+  else if (e.target.id === "tabs-add-zoom") {
+      callOnActiveTab((tab) => {
+        chrome.tabs.getZoom(tab.id, function(zoomFactor){
+            var newZoomFactor = zoomFactor + 0.2;
```

Comment:
It would be nice to define constants for things like this.

e.g. 
const ZOOM_INCREMENT = 0.2;
const MAX_ZOOM = 3;
const MIN_ZOOM = 0.3;
const DEFAULT_ZOOM = 1;